tests/manage/pv_services/test_rwo_pvc_fencing_unfencing.py

```diff
@@ -0,0 +1,983 @@
+import logging
+import pytest
+import random
+from concurrent.futures import ThreadPoolExecutor
+from time import sleep
+
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import (
+    ignore_leftovers, ManageTest, tier4, tier4a, tier4b, tier4c
+)
+from ocs_ci.ocs import constants, machine, node, ocp
+from ocs_ci.ocs.cluster import CephCluster
+from ocs_ci.ocs.exceptions import ResourceWrongStatusException, UnexpectedBehaviour
+from ocs_ci.ocs.resources import pod
+from ocs_ci.utility.retry import retry
+from ocs_ci.utility.utils import ceph_health_check, get_az_count
+from tests import disruption_helpers, helpers
+
+logger = logging.getLogger(__name__)
+
+
+@tier4
```

Comment:
LGTM. Just one comment.
Add a sub-tier to pick for regular tier run.

Reply:
Yes, the tests are distributed in three sub-tiers, those sub tiers markers are present at the top of the test functions